ansible/roles/operator-pipeline/templates/openshift/tasks/git-clone.yml

```diff
@@ -0,0 +1,213 @@
+apiVersion: tekton.dev/v1beta1
```

Comment:
Why aren't we using the ready-made task here?

Reply:
I tried it in the first version, but k8s can't deal with remote resources so I pull the task into repo directly.